### PR TITLE
feat(elements): Add coinbase on elements package

### PR DESCRIPTION
**Pull request by EmmanouelaPothitou**
## Description
Add coinbase on elements package

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

**Review by tmilewski — APPROVED** [review]

**Comment by tmilewski** [comment]
@EmmanouelaPothitou Looks like there's some typing issues but, otherwise, 👍.

File: .changeset/polite-apricots-search.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@clerk/elements": patch
+---
+
+Add coinbase wallet  on elements package
```
Comment (dstaley) [inline .changeset/polite-apricots-search.md:5]:
```suggestion
Add support for sign in and sign up with Coinbase
```

**Comment by dstaley** [comment]
Just dropping a note that this PR is dependent on the functionality in https://github.com/clerk/javascript/pull/3940, so until that's merged CI for this one will fail.

**Review by dstaley — APPROVED** [review]
Assuming the API for the Coinbase flow doesn't change, this looks good to me as long as you update the changelog entry to be a bit more descriptive!

**Pull request merged**